### ISSUE TITLE
Character Sheet Suggestion

### ISSUE THREAD

**Issue by raihje** [openlegend/core-rules#56]
Right now, theoretically any player can use any bane/boon + weapons. Even if they don't, as they level they'll potentially have a pretty long list of actions. The 3 Defenses could be one line at the top, with just a bit of text in the box saying X + X + ____ I don't think it's necessary to write down the same attribute more than once on the sheet. Hit Points, Speed, Legend Pts, and Wealth could be the next line  with Actions having more real estate beneath that to make more efficient use of space. The Feats/Perks looks alright, since Feats are obtained slower and by lv10 they likely still won't fill all those lines unless they aren't taking multiple tiers. It's artfully displayed at the moment and it looks good, but in my opinion if it was more space efficient it'd save players from needing more than the one sheet for basic information and maybe one additional piece of lined paper / whatever for backstory, description, etc.

**Comment by thefailking** [comment]
yet totally but I can envision the character sheet being much more simplified for example the cogent role play system is super simple and I can imagine open legend having a similar sheet like the one in cogent.

**Comment by brianfeister** [comment]
Hey @raihje that's a bit tough to grok from a text-based explanation. Can you please arrange that visually, to the best of your ability. It's hard to process otherwise, especially for me, being a very visual person moreso than verbal.

**Comment by thefailking** [comment]
![cogent](https://cloud.githubusercontent.com/assets/5202710/19780587/a20b508c-9c7d-11e6-96b0-cbc38781b894.PNG)
this is what it looks like for those who haven't seen cogent character sheets before

**Comment by DukeAnax** [comment]
This is an example on how it could be optimised, without taking much from the visual:
![192536neu](https://cloud.githubusercontent.com/assets/23003774/19782031/4d9d651c-9c8c-11e6-96ab-b7e03c77853a.jpg)

**Comment by brianfeister** [comment]
Alrighty. I'm not quite satisfied with these options, however, they are a **big** help in communicating what you folks are looking for. Thanks!

**Comment by brianfeister** [comment]
I'll try to come up with a new proposal and post it here.

**Comment by raihje** [comment]
I think the keywords are space efficient. The image above speaks for itself, but I would put the three defense types next to each other at the top. Scale hitpoints down (since it really will only hold 2, 2 digit numbers) so that the hitpoints and defense types form a clean "line".
Wealth, Legend Points, and Speed can each be small (height wise) and make their own "line" beneath the defense types and hit points. In my opinion this would look much cleaner, and give more room for the stuff that really takes up space like actions, potentially inventory.

**Comment by jjnich** [comment]
I like the idea of having inventory on the sheet. It should have 20 lines though (as you can hold a max of 20 items).

**Comment by brianfeister** [comment]
Indeed. I'm pretty sure this is going to become 2 pages, as much as I wanted to avoid it.

**Comment by Groumy** [comment]
Question, to create a little more real estate on the sheet, do we really need a box to keep track how much point were invested on every ability ?

I mean it's kind of an easy to get information, it's in the rules in a neat table and it's an information that doesn't help at all when you actually play. It just help to create the character.

I know it's not a lot of extra space but it's a start.

**Comment by raihje** [comment]
You could also put HP as a smaller thing underneath the attributes flag, then move speed wealth and legend points to where hitpoints currently is.
The total points will always be your level * 3 + 6. It's super simple to do in your head, and as you level up or get xp it'll be easy to just figure out on the fly, since your XP/Level is written on the sheet already. Points spent makes some sense, but you could do an attribute points / feat points line in the description. I'll see if I can come up with a decent image example.

**Comment by raihje** [comment]
@brianfeister @istabosz @DukeAnax Here's something I put together in Libreoffice. If's not pretty, but it should give you guys a good idea. I made sure to leave a good chunk of space for styling, this is just so people can visualize the layout I have in mind. [Link to PDF](https://drive.google.com/file/d/0B2SlomurczYsdlhBc0x4d3NiQjg/view?usp=sharing)

**Comment by raihje** [comment]
@brianfeister @istabosz @DukeAnax Here is my idea for the layout. I put this together in LibreOffice, which I'm not particularly good at using, but this is just so you guys can visualize the layout I have in mind. I left open space for styling / shuffling stuff. It's set to US Letter, which is what I imagine the majority of people will be printing with. It's got a quarter inch margin and space between the columns.
[Link to PDF](https://drive.google.com/file/d/0B2SlomurczYsNmRFLTdvTXhFWVE/view?usp=sharing)

**Comment by brianfeister** [comment]
I see what you mean. I think we could probably go 50/50 on feats vs. Inventory, since inventory starts big but doesn't change and feats can get pretty long. I appreciate the proposal, I'll see what I can do about revising the sheet. No need to trouble yourself too much, I think you've given me plenty to go off of.

**Comment by raihje** [comment]
As long as the Inventory has 20 slots, it can be shrunk down I guess. It's not like most items have particularly long names, but it could come in handy for writing  their properties.

**Comment by ucffool** [comment]
Also, in reference to @raihje 's beautiful sheet, two things:
1) Current hit points is usually the most changed box on the page and thus why it is usually a bit larger.
2) The style is perfect for fantasy, but since we are aiming not to be exclusionary to other genres, it'd be awesome if the final version (layout) had different flourishes/styling for other genres. Little thing that would make a ton of folks happy.

**Comment by brianfeister** [comment]
@ucffool there will ultimately be a separate character sheet for non-fantasy. I haven't quite been convinced by this proposal. Some ideas are worth keeping and others not so I just need to do my own version.

**Comment by raihje** [comment]
Well if you'd like me to take another crack at it I can. I work full time now so that's kind of why I disappeared for a bit but evenings (for a few hours) and weekends I'm more available now.
